src/zenml/integrations/gcp/step_operators/vertex_step_operator.py

```diff
@@ -55,14 +57,14 @@ class VertexStepOperator(BaseStepOperator, GoogleCredentialsMixin):
 
     Attributes:
         region: Region name, e.g., `europe-west1`.
-        project: [Optional] GCP project name. If left None, inferred from the
+        project: GCP project name. If left None, inferred from the
             environment.
-        accelerator_type: [Optional] Accelerator type from list: https://cloud.google.com/vertex-ai/docs/reference/rest/v1/MachineSpec#AcceleratorType
-        accelerator_count: [Optional] Defines number of accelerators to be
+        accelerator_type: Accelerator type from list: https://cloud.google.com/vertex-ai/docs/reference/rest/v1/MachineSpec#AcceleratorType
+        accelerator_count: Defines number of accelerators to be
             used for the job.
-        machine_type: [Optional] Machine type specified here: https://cloud.google.com/vertex-ai/docs/training/configure-compute#machine-types
-        base_image: [Optional] Base image for building the custom job container.
-        encryption_spec_key_name: [Optional]: Encryption spec key name.
+        machine_type: Machine type specified here: https://cloud.google.com/vertex-ai/docs/training/configure-compute#machine-types
+        base_image: Base image for building the custom job container.
+        encryption_spec_key_name: Encryption spec key name.
```

Comment:
Why do these stop being optional, are we replacing them with sane defaults?

Reply:
They stay optional, but it's already specified in the type annotation and shouldn't be in the docstring.

See for example here for the duplicated `Optional`:
https://apidocs.zenml.io/0.11.0/api_docs/integrations/#zenml.integrations.gcp.step_operators.vertex_step_operator.VertexStepOperator